specification/_json_spec/capabilities.json

```diff
@@ -1,7 +1,7 @@
 {
   "capabilities": {
     "documentation": {
-      "url": "https://www.elastic.co/guide/en/elasticsearch/reference/master/capabilities.html",
+      "url": "https://github.com/elastic/elasticsearch/blob/main/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/README.asciidoc#require-or-skip-api-capabilities",
```

Comment:
That one is weird, I would double check the Elasticsearch PR to make sure this was intentional.

Reply:
It looks weird but its actually on purpose, the documentation link doesn't exist.
I'll merge as is.